framework/src/wifi_manager/wifi_manager_lwnl.c

```diff
@@ -112,6 +112,7 @@ wifi_utils_result_e wifi_utils_scan_ap(void *arg)
 	if (_send_msg(&msg) < 0) {
 		return WIFI_UTILS_FAIL;
 	}
+	lldbg("[pkbuild] lwnl %d\n", res);
```

Comment:
All other functions are using `NET_LOGE`. How about using `NET_LOGE` instead of `lldbg`?